src/components/LoadingBar.tsx

```diff
@@ -0,0 +1,85 @@
+import React, {useEffect} from 'react';
+import Animated, {cancelAnimation, Easing, runOnJS, useAnimatedStyle, useSharedValue, withDelay, withRepeat, withSequence, withTiming} from 'react-native-reanimated';
+import useThemeStyles from '@hooks/useThemeStyles';
+import CONST from '@src/CONST';
+
+type LoadingBarProps = {
+    /** Whether to show the loading bar with fade out animation */
```

Comment:
Can this be a regular comment (// instead of /**) Also, you can remove the "with fade out animation". So the comment should be "Whether or not to show the loading bar"